tidysic/exceptions.py

```diff
@@ -0,0 +1,83 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+
+from tidysic.file.taggable import Taggable
+from tidysic.file.tagged_file import TaggedFile
+from tidysic.logger import Message, String, Text
+
+
+class TidysicException(Exception, ABC):
+    @abstractmethod
+    def get_error_message(self) -> Message:
+        """
+        Returns an error message friendly to the logger module.
+        """
+        pass
```

Comment:
triple dot is the trendy new `pass` these days
```suggestion
        ...
```

Reply:
Wow I did not know. I gotta get with the times.

Reply:
![cool kids](https://media.giphy.com/media/Lm628LV0b6UNIULJPg/giphy.gif)